noir/noir-repo/compiler/noirc_frontend/src/hir/def_collector/dc_crate.rs

```diff
@@ -153,7 +160,7 @@ pub struct CollectedItems {
 impl CollectedItems {
     pub fn is_empty(&self) -> bool {
         self.functions.is_empty()
-            && self.types.is_empty()
+            && self.structs.is_empty()
             && self.type_aliases.is_empty()
```

Comment:
@jfecher looks like this didn't get updated to reflect enums

Reply:
Unless I'm reading it wrong, it looks backward to me? Why are we undoing the enum changes in the sync?